### PR TITLE
Portal maintenace mode

### PR DESCRIPTION
**Pull request by joroca**
- New feature for setting up portal on maintenance mode.
- Sessions save on mongodb (Requires pymongo==2.9 and beaker==1.9.0).
- Online users list in maintenace page.
- Showing a message to logged users warning them for imminent change to maintenance mode.

### CODE REVIEW

File: gecoscc/userdb.py [hunk 1]
```diff
@@ -170,4 +170,7 @@ def get_user(request):
 def get_groups(userid, request):
     ## TODO
     # Not Implemented
-    return ['logged']
+    user = request.userdb.get_user(userid)
+    if user.get('is_superuser',None):
+        return ['logged', 'g:maintenance']
+    return ['logger']
```
Comment (alfem) [inline gecoscc/userdb.py]: "logger" or "logged"?

File: gecoscc/views/settings.py [hunk 2]
```diff
@@ -28,6 +28,9 @@
 from bson.objectid import ObjectId
 logger = logging.getLogger(__name__)
 
+
```
Comment (alfem) [inline gecoscc/views/settings.py]: What is this for?

Comment (joroca) [reply gecoscc/views/settings.py]: This settings, which are not set in the view, are saved in MongoDB collection.

File: gecoscc/userdb.py (hunk 1 above)
Comment (joroca) [reply gecoscc/userdb.py]: 'logged'. Corrected.

**Pull request merged**